### DISCUSSION TITLE
Error: Instance Start Failed

### DISCUSSION THREAD

**Discussion by apfeltasche** [FlowFuse/flowfuse#2706] [Q&A]
Hi guys, 
i got a error, doas anyone know how to solve it?

Error: Instance Start Failed
Show Error
{ "error": { "code": "unexpected_error", "message": "Failed to start project 83b1dfbd-b7a5-4c56-911c-b69b6f09cbfc: TypeError: Cannot read properties of undefined (reading 'Name')" } }

**Comment by hardillb** [comment]
Please include the full Node-RED logs and details of all the versions you are using on what platform

**Reply by apfeltasche** [reply]
Error Message:
TypeError: Cannot read properties of undefined (reading 'Name') 
forge_1             | [2023-09-26T14:35:00.730Z] INFO: AUDIT: event: project.started, type: project, id: 3f1029ec-ce03-4354-a622-4e64079ca586 
forge_1             | [2023-09-26T14:35:37.631Z] ERROR: Failed to start project a61ad5e5-5a54-46fe-b3a5-b2be08a7e1b4: TypeError: Cannot read properties of undefined (reading 'Name') 
forge_1             | [2023-09-26T14:35:37.633Z] INFO: AUDIT: event: project.created, type: project, id: a61ad5e5-5a54-46fe-b3a5-b2be08a7e1b4 
forge_1             | [2023-09-26T14:35:37.637Z] ERROR: AUDIT: event: project.start-failed, type: project, id: a61ad5e5-5a54-46fe-b3a5-b2be08a7e1b4, Failed to start project a61ad5e5-5a54-46fe-b3a5-b2be08a7e1b4: TypeError: Cannot read properties of undefined (reading 'Name') 
forge_1             | [2023-09-26T14:35:37.641Z] INFO: AUDIT: event: project.created, type: team, id: 1 
forge_1             | [2023-09-26T14:35:58.567Z] INFO: AUDIT: event: project.deleted, type: team, id: 1 
forge_1             | [2023-09-26T14:35:58.569Z] INFO: AUDIT: event: project.deleted, type: project, id: a61ad5e5-5a54-46fe-b3a5-b2be08a7e1b4 
forge_1             | [2023-09-26T14:36:47.187Z] INFO: AUDIT: event: team.settings.updated, type: team, id: 1 
forge_1             | [2023-09-26T14:36:53.643Z] ERROR: Failed to start project 3f1029ec-ce03-4354-a622-4e64079ca586: TypeError: Cannot read properties of undefined (reading 'Name') 
forge_1             | [2023-09-26T14:36:53.646Z] ERROR: AUDIT: event: project.start-failed, type: project, id: 3f1029ec-ce03-4354-a622-4e64079ca586, Failed to start project 3f1029ec-ce03-4354-a622-4e64079ca586: TypeError: Cannot read properties of undefined (reading 'Name')

**Reply by apfeltasche** [reply]
[docker-compose.txt](https://github.com/flowforge/flowforge/files/12735196/docker-compose.txt)

**Reply by hardillb** [reply]
What version of docker-compose are you using?

Also are you running in a Debian host machine?

FlowFuse requires docker-compose v2

**Reply by apfeltasche** [reply]
Ok, that may be the problem - its v1.

I will try it out!

**Reply by apfeltasche** [reply]
Unfortunatly its the same error with compose V2.21.0.

**Reply by hardillb** [reply]
You will need to remove everything and start again with docker-compose v2

So
- remove all instances
- run `docker-compose -p flowforge down`
- ensure that the flowforge network is removed (`docker network list`)
- restart with `docker-compose -p flowforge up`
-  check that the network has the right labels `docker network inspect flowforge`
